### PR TITLE
Bump bounds for fpco/stackage #2140.

### PR DESCRIPTION
**Pull request by lwm**
Following from https://github.com/hvr/packunused/issues/28.

I think that should do it ...

### CODE REVIEW

**Comment by hvr** [comment]
HSE's major version bumps quite often signal significantly breaking API changes... :-)

**Comment by lwm** [comment]
Ah yes, I see ;). 

I've moved to HSE 1.18.2, which should be fine for the >1.18 bounds error. 

Once later versions of Stackage cover later versions of HSE, I'll move back to Stack LTS.

File: packunused.cabal [hunk 1]
```diff
@@ -79,9 +79,9 @@ executable packunused
   build-depends:
     base                 >=4.5  && <4.10,
     Cabal                >=1.24 && <1.25,
-    optparse-applicative >=0.8  && <0.13,
+    optparse-applicative >=0.8  && <0.14,
     directory            >=1.1  && <1.3,
     filepath             >=1.3  && <1.5,
-    haskell-src-exts     >=1.14 && <1.18,
+    haskell-src-exts     ==1.18.2,
```
Comment (hvr) [inline packunused.cabal]: tbh, that's overkill; please use `>= 1.18.2 && < 1.19`

Comment (lwm) [inline packunused.cabal]: Oh, thought you meant to fix on a single version. Will amend when I get a minute :+1:

**Comment by igrep** [comment]
@hvr I'm interested in trying this package with GHC.
Is this branch ready to merge?

**Comment by hvr** [comment]
@igrep I think so... I just merged it (@lwm thanks!)

**Comment by lwm** [comment]
Great! Thanks :beers:

**Comment by lwm** [comment]
Great! Thanks :beers:

**Pull request merged**